monai/transforms/io/array.py

```diff
@@ -31,38 +32,53 @@
 
 class LoadImage(Transform):
     """
-    Load image file or files from provided path based on reader, default reader is ITK.
-    All the supported image formats of ITK:
-    https://github.com/InsightSoftwareConsortium/ITK/tree/master/Modules/IO
+    Load image file or files from provided path based on reader, default reader is Nibabel.
     Automatically choose readers based on the supported suffixes and in below order:
     - User specified reader at runtime when call this loader.
     - Registered readers from the first to the last in list.
-    - Default ITK reader.
+    - Default Nibabel reader.
 
     """
 
     def __init__(
         self,
-        reader: Optional[ImageReader] = None,
+        reader: Optional[Union[ImageReader, str]] = None,
         image_only: bool = False,
         dtype: np.dtype = np.float32,
+        *args,
+        **kwargs,
     ) -> None:
         """
         Args:
             reader: register reader to load image file and meta data, if None, still can register readers
-                at runtime or use the default ITK reader.
+                at runtime or use the default NIbabel reader. If a string of reader name provided, will construct
+                a reader object with the `*args` and `**kwargs` parameters, supported reader name: "NibabelReader",
+                "PILReader", "ITKReader", "NumpyReader"
             image_only: if True return only the image volume, otherwise return image data array and header dict.
             dtype: if not None convert the loaded image to this data type.
+            args: additional parameters for reader if providing a reader name.
+            kwargs: additional parameters for reader if providing a reader name.
 
         Note:
             The transform returns image data array if `image_only` is True,
             or a tuple of two elements containing the data array, and the meta data in a dict format otherwise.
 
         """
-        self.default_reader: ITKReader = ITKReader()
+        self.default_reader: NibabelReader = NibabelReader()
         self.readers: List[ImageReader] = list()
         if reader is not None:
+            if isinstance(reader, str):
+                supported_readers = {
```

Comment:
could be combined with `self.readers`?

Reply:
Hi @wyli ,

`self.readers` is used to record all the readers in the `LoadImage` transform, we can register readers to the list and the **order of the list** maps to the reader priorities.
And you can set 2 `ITKReader` with different parameters.

Thanks.

Reply:
ok, I think by default if `LoadImage(reader=None)` it should automatically try a set of different reader classes, that was our original requirement.

Reply:
We don't have this logic so far. Acutally, it's not very easy to achieve this requirement, we need to define the default reader priorities list for every kind of image format.
Of course, it sounds like a good idea, now we only use `NibabelReader` as default reader, we can try to enhance it later.

Thanks.

Reply:
the idea of feature #909 is to create a file format agnostic wrapper. as a starting point, could we support the logic of: 

```
if nifti file try nibabel
elif jpg/png try pillow
elif npy try loadnumpy
else try itk
```
if you don't have time to implement it I think we should postpone this feature

Reply:
OK, I will try to make the default reader automatically select reader type.
Thanks.